compiler/ast/src/input/input_ast.rs

```diff
@@ -31,6 +33,34 @@ pub struct ParsedInputFile {
     pub sections: Vec<Section>,
 }
 
+impl ParsedInputFile {
```

Comment:
Could we change to `InputAst` to be more consistent?
```suggestion
impl InputAst {
```